data/ships.txt

```diff
@@ -177,6 +177,109 @@ ship "Arrow"
 
 
 
+ship "Auxiliary"
+	sprite "ship/auxiliaryh"
+	thumbnail "thumbnail/auxiliaryh"
+	attributes
+		category "Transport"
+		licenses
+			Navy
+			Auxiliary
+		"cost" 16720000
```

Comment:
16.72 M? For comparison, the Bactrian's base cost is 17.6 M, the Carrier's 15.2 M, and the Cruiser's 11.2 M.